wordpress-mobile/WordPress-iOS
# [Gravatar] - Design updates on the informative Gravatar UI

Fixes https://github.com/wordpress-mobile/WordPress-iOS/issues/22551

Continuation of PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/22563 

We are doing a design update on the informative Gravatar UI. Basically we want it to look like the this part in the Help & Support page:

<img width=300 src="https://github.com/wordpress-mobile/WordPress-iOS/assets/5032900/6ff972fb-26bc-4083-8c13-a8a23fb0820f">

**Jetpack:**

<table>
  <tr>
    <td><img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/5032900/29dab9af-df11-4473-9862-04218009183a" width=300/></td>
    <td><img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/5032900/d63052a6-0838-4a6a-807b-8b572611a5c8" width=300/></td>
  </tr>
</table>

**WordPress:**

<table>
  <tr>
    <td><img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/5032900/6fc688f4-dbc0-4f92-b182-2a0b24ea3529" width=300/></td>
    <td><img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/5032900/4efb2b5c-731d-4290-a160-79d667de3df0" width=300/></td>
  </tr>
</table>

* Removes the logos that was added in https://github.com/wordpress-mobile/WordPress-iOS/pull/22563 
* Adds new logos
* Reuses the existing cells in the app 

To test:

Go to the profile tab > My Profile
Observe: There's a new informative UI as demonstrated above ☝️ Check if everything looks fine.

## Regression Notes

1. Potential unintended areas of impact

Same UI in Help & Support

3. What I did to test those areas of impact (or what existing automated tests I relied on)

"Me Tab > Help & Support > Top section" should look and work fine and the link there should also work:

<img width=300 src="https://github.com/wordpress-mobile/WordPress-iOS/assets/5032900/6ff972fb-26bc-4083-8c13-a8a23fb0820f">

4. What automated tests I added (or what prevented me from doing so)

PR submission checklist:


Testing checklist:
